src/components/Forms/PostComment.jsx

```diff
@@ -0,0 +1,61 @@
+import { useContext, useEffect} from "react";
+import { useState } from "react";
+import { userContext } from "../../Context/UserProvider";
+import axios from "axios";
+import ErrorPopup from "../ErrorPopup";
+
+export default function PostComment({ article_id}) {
+    const { user, setUser } = useContext(userContext);
+    const [errStatus, setErrStatus] = useState(null);
+    const [comment, setComment] = useState("");
+
+    function popupErrHandler() {
+        setErrStatus(null);
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault();
+        const commentData = {body:comment, article_id:article_id, user:user.username} 
+        axios.post(`https://nc-news-u90o.onrender.com/api/articles/${article_id}/comments`, commentData)
```

Comment:
Ideally this function should be in a nice utils file with all your other api calls